default.yml

```diff
@@ -46,17 +41,6 @@ Layout/MultilineOperationIndentation:
 Lint/SuppressedException:
   Enabled: false
 
-Layout/SpaceInsideBlockBraces:
-  # The space here provides no real gain in readability while consuming
-  # horizontal space that could be used for a better parameter name.
-  # Also {| differentiates better from a hash than { | does.
-  SpaceBeforeBlockParameters: false
```

Comment:
I disagree that the space provides no readability gain.

```ruby
# no space before block
my_method.map {|item| item.to_i }

# space before block
my_method.map { |item| item.to_i }
```

Reply:
No preference here

Reply:
This one I have seen both ways in our code, and would much rather have space. It looks cleaner, it is easier to read, and again consistent with the community. I also believe our esLinter does it this way too.

Reply:
Same as [1]

Reply:
I prefer the space but I'm good either way since this is an auto-fix.